flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/source/OracleSourceITCase.java

```diff
@@ -146,7 +146,8 @@ private void testOracleParallelSource(
                                 + " 'table-name' = '%s',"
                                 + " 'scan.incremental.snapshot.enabled' = 'false',"
                                 + " 'debezium.log.mining.strategy' = 'online_catalog',"
-                                + " 'debezium.log.mining.continuous.mine' = 'true'"
+                                + " 'debezium.log.mining.continuous.mine' = 'true',"
+                                + " 'debezium.database.history.store.only.captured.tables.ddl' = 'true'"
```

Comment:
We can remove config `[database.history.store.only.captured.tables.ddl](https://debezium.io/documentation/reference/1.9/connectors/oracle.html#oracle-property-database-history-store-only-captured-tables-ddl)` in test case.

Reply:
But in this case, tests with `parallelismSnapshot: false` will take much longer to run because of capturing structure of system APEX tables. I think, that we don't need to monitor them during tests.

Reply:
Ok, It can speed up unit test. I got it.